content/documentation/bootstrap-nodes.mdx

```diff
@@ -3,7 +3,7 @@ title: Bootstrap nodes
 seo: Bootstrap Nodes | Get Started
 ---
 
-Bootstrap nodes are special nodes that you connect to to get information about new peers. By default there is an Iron Fish foundation provided bootstrap node for each network.
+Bootstrap nodes are special nodes that you connect to get information about new peers. By default there is an Iron Fish foundation provided bootstrap node for each network.
```

Comment:
This is correct as-is. If you wanted to make it more clear, you could say "Bootstrap nodes are special nodes that you connect to in order to get information about new peers."